releasenotes/notes/log-manager-18fb90743f423087.yaml

```diff
@@ -0,0 +1,6 @@
+features:
```

Comment:
I think it is important to add that there is a change in behavior when reporting files to beaker.   The logs will not be written until the completion of the task.